crypto/init.c

```diff
@@ -44,6 +44,7 @@ struct ossl_init_stop_st {
 };
 
 static OPENSSL_INIT_STOP *stop_handlers = NULL;
+static CRYPTO_RWLOCK *optsdone_lock = NULL;
 static CRYPTO_RWLOCK *init_lock = NULL;
```

Comment:
I think we need some comments here to explain the two different locks and what they should be used for.

Reply:
Better now?